src/test/user/login/get.controller.test.js

```diff
@@ -43,6 +43,7 @@ describe('User Login Get Controller', () => {
   it('should render if referer but no user object in session', async () => {
     req.headers.referer = '/example';
     const cookie = new CookieModel(req);
+    delete req.session.u;
```

Comment:
Controller code never had the situation of no user defined tested.

It is actually a contrived scenario, as a user object is always present due to the CookieModel creating it, even with null values when empty.